parser.go

```diff
@@ -159,6 +159,7 @@ func (parser *Parser) ParseAPI(searchDir string, mainAPIFile string) error {
 
 	var t depth.Tree
 	t.ResolveInternal = true
+	t.MaxDepth = parseDepth
```

Comment:
Can you help move the three lines into ``` if parser.ParseDependency {
}``` ?

Reply:
done

Reply:
thks